plugins/pipelines/create.js

```diff
@@ -145,7 +139,8 @@ module.exports = () => ({
                                                                 value: privateDeployKeyB64,
                                                                 allowInPR: false
                                                             });
-                                                        });
+                                                        })
+                                                        .catch(err => reply(boom.boomify(err)));
```

Comment:
won't this still execute rest of the `Promise` chain ?
I thought was this catch is not required, and let the `err` propagate to the final `catch` at https://github.com/screwdriver-cd/screwdriver/pull/2157/commits/423742205e5363e72b09e8ff630d383d27c2d877#diff-1eb6375f360812e82460d42f926123b1R170